yambol/yambol/parser.py

```
@@ -0,0 +1,43 @@
+    t = default_values | t
+    db_type = db_types.Type(**t)
+    types.append(db_type)
+  return types
+  
+def load_tables(data: dict) -> typing.List[db_types.Table]:
+  tables = []
+  default_values = db_types.Table.__annotations__
+  for t in data['tables']:
+    t = default_values | t
+    db_table = db_types.Table(**t)
+    for key in t.keys():
```

Ruff (0.8.2)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)